src/OpenTelemetry.SemanticConventions/scripts/semantic-conventions/generate.ps1

```diff
@@ -3,47 +3,64 @@ $SCRIPT_DIR=$PSScriptRoot
 $ROOT_DIR="${SCRIPT_DIR}/../../"
 
 # freeze the spec & generator tools versions to make SemanticAttributes generation reproducible
-$SPEC_VERSION="v1.13.0"
+$SPEC_VERSION="v1.23.1"
+$SPEC_VERSION_ESCAPED="v1_23_1_Experimental"
 $SCHEMA_URL="https://opentelemetry.io/schemas/$SPEC_VERSION"
-$GENERATOR_VERSION="0.14.0"
+$GENERATOR_VERSION="foo14"
 
 Set-Location $SCRIPT_DIR
 
-Remove-Item -r -fo opentelemetry-specification
-mkdir opentelemetry-specification
-Set-Location opentelemetry-specification
+Remove-Item -r -fo semantic-conventions
+mkdir semantic-conventions
+Set-Location semantic-conventions
 
 git init
-git remote add origin https://github.com/open-telemetry/opentelemetry-specification.git
+git remote add origin https://github.com/open-telemetry/semantic-conventions.git
 git fetch origin $SPEC_VERSION
 git reset --hard FETCH_HEAD
 Set-Location ${SCRIPT_DIR}
 
+# stable attributes
 docker run --rm `
-  -v ${SCRIPT_DIR}/opentelemetry-specification/semantic_conventions/trace:/source `
+  -v ${SCRIPT_DIR}/semantic-conventions/model:/source `
   -v ${SCRIPT_DIR}/templates:/templates `
-  -v ${ROOT_DIR}/Trace:/output `
-  otel/semconvgen:$GENERATOR_VERSION `
-  -f /source code `
-  --template /templates/SemanticConventions.cs.j2 `
-  --output /output/TraceSemanticConventions.cs `
+  -v ${ROOT_DIR}/SemanticConventions:/output `
+  semconvgen:$GENERATOR_VERSION `
+  --yaml-root /source code `
+  --template /templates/Attributes.cs.j2 `
+  --output /output/Attributes.cs `
   --trim-whitespace `
-  -D class=TraceSemanticConventions `
-  -D schemaUrl=$SCHEMA_URL `
-  -D pkg=OpenTelemetry.Trace
+  --file-per-group root_namespace `
+  -D filter=is_stable `
+  -D pkg=OpenTelemetry.SemanticConventions
 
+#experimental attributes
```

Comment:
it's easy to filter stable/experimental and move them into different artifacts, use different templates, add `[Experimental]` attribute, etc